Vagrantfile

```diff
@@ -122,15 +126,34 @@ Vagrant.configure(2) do |config|
     config.vm.synced_folder ".", "/vagrant/commissaire", type: "sshfs"
     config.vm.synced_folder "../commissaire-http", "/vagrant/commissaire-http",  type: "sshfs"
     config.vm.synced_folder "../commissaire-service", "/vagrant/commissaire-service", type: "sshfs"
-    commissaire.vm.provision "shell", inline: <<-SHELL
-      echo "==> Setting hostname"
+    commissaire.vm.provision "shell", env:
+      {"ETCD_SERVER" => "192.168.152.101", "ETCD_PORT" => "2379"}, inline: <<-SHELL
+      echo "===> Setting hostname"
       sudo hostnamectl set-hostname commissaire
+
+      echo "===> Setting SELinux to Permissive mode (for custodia.socket)"
+      sudo setenforce Permissive
+
       echo "===> Updating the system"
       sudo dnf update -y
+
       echo "===> Installing OS dependencies"
       sudo dnf install -y --setopt=tsflags=nodocs rsync openssh-clients redhat-rpm-config python3-virtualenv gcc libffi-devel openssl-devel git nfs-utils etcd
+
+      echo "===> Installing custodia (with etcd module) from f26"
```

Comment:
Hmm, okay.  I was trying to avoid `pip` and install all dependencies from rpm packages (with RHEL 8 in mind).  Though, even the F26 `custodia` package is a bit old.